src/main/java/com/google/devtools/build/lib/analysis/RunfilesSupport.java

```diff
@@ -444,4 +446,14 @@ private static CommandLine computeArgs(
         ruleContext.getExpander().withDataLocations().tokenized("args"),
         additionalArgs);
   }
+
+  /** Provided the path of the runfiles directory returns the path of the input manifest. */
```

Comment:
The comment just spells out the function signature. Can you make it more useful? Explaining the input manifest's use could be helpful. How about something along these lines:

> Returns the input manifest for runfiles creation.
> The input manifest lists pairs of paths: execroot-relative paths of the runfiles with the runfiles-root-relative path they should be mapped to.

Reply:
done (inside google)